renku/core/commands/cluster.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017-2021 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku cluster command."""
+
+from pathlib import Path
+from subprocess import PIPE, STDOUT, run
+
+from jinja2 import Template
+
+from renku.core.incubation.command import Command
+from renku.core.management.config import CONFIG_LOCAL_PATH
+from renku.core.commands.docker import detect_registry_url
+
+CONFIG_SECTION = "cluster"
+IMAGE_SHA = 7
+
+
+def _prepare_cluster_config(client, sbatch_options, gitlab_token):
```

Comment:
With recent changes to code, `client` needs to be injected using `@inject`, see e.g. https://github.com/SwissDataScienceCenter/renku-python/blob/master/renku/core/commands/dataset.py#L76-L81